docs/datasets/data-anonymization/index.md

```diff
@@ -0,0 +1,221 @@
+# Rosbag2 Anonymizer
+
+## Overview
+
+Autoware provides a tool to anonymize ROS 2 bag files.
+This tool is useful when you want to share your data with Autoware community but want to keep the privacy of the data.
+
+With this tool you can blur any object (faces, license plates, etc.) in your bag files, and you can get a new bag file
+with the blurred images.
+
+## Installation
+
+### Clone the repository
+
+```bash
+git clone https://github.com/autowarefoundation/autoware_rosbag2_anonymizer.git
+```
```

Comment:
After clonning the repository user need to go to downloaded folder 
![image](https://github.com/autowarefoundation/autoware-documentation/assets/112626902/58223183-1d70-46e8-b697-2f29962f4e08)
For further downloads of pretrained models need to be in the autoware_rosbag2_anonymizer folder.